parkings/migrations/0015_fill_normalized_reg_nums.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.db.models import Q
+
+from ..models import Parking
+
+
+def fill_normalized_reg_nums(apps, schema_editor):
+    parking_model = apps.get_model('parkings', 'Parking')
+    parkings_to_process = parking_model.objects.filter(
+        Q(normalized_reg_num=None) | Q(normalized_reg_num=''))
+    for parking in parkings_to_process:
+        parking.normalized_reg_num = Parking.normalize_reg_num(
+            parking.registration_number)
+        parking.save(update_fields=['normalized_reg_num'])
+
+
+def noop(apps, schema_editor):
+    pass
```

Comment:
There is `RunPython.noop` for this purpose :)

Reply:
> The existing initial internal API was meant for this exact purpose, so it could and should be removed now then :)

OK then. I didn't want to change it, because I wasn't able to find any documentation where it's used. Bad luck, since it would have been what was needed.

Reply:
Thanks, will use that then.